jupyterhub/jupyterhub/overlays/build/jupyterhub-imagestream.yaml

```diff
@@ -5,4 +5,4 @@ metadata:
   name: jupyterhub
 spec:
   lookupPolicy:
-    local: true
+    local: true
```

Comment:
Missing new line

Reply:
Ok, I've added newlines to all the files